web/src/user/UserInterface.ts

```diff
@@ -364,4 +273,202 @@ export class UserInterface extends Interface {
             </div>
         </ak-locale-context>`;
     }
+
+    renderApiDrawer() {
+        if (!this.uiConfig.enabledFeatures.apiDrawer) {
+            return nothing;
+        }
+
+        const onClick = (ev: Event) => {
```

Comment:
The current version has _two_ handlers for EVENT_API_DRAWER_TOGGLE, one attached to `UserInterface`, another attached to `window`.  No sense in keeping both; this eliminates one of them.

Reply:
would probably rename this to `renderApiDrawerTrigger` since it doesn't render the actual drawer